src/components/o-modal/_preview.html

```diff
@@ -1,31 +1,31 @@
 <button type="button" onclick="openModal()">Open modal</button>
 
-<axa-modal class="axa-modal-preview">
+<!-- The style attribute is necessary to avoid the FOUC problem -->
+<axa-modal class="axa-modal-preview" style="display:none;">
```

Comment:
I would use a modifier instead of s style attr `axa-modal-preview--hidden` for example

Reply:
Be aware that only inline style will be available before the component has been downloaded.
If you add a modifier within the `<axa-modal>` CSS, this CSS will only be available after component download and will not be able to fix FOUC, because it is loaded too late

Reply:
ahhh yes good point sorry